runtime/crab/src/pallets/bridge_grandpa.rs

```diff
@@ -26,7 +26,7 @@ pub type PolkadotHeadersToKeep = ConstU32<500>;
 impl pallet_bridge_grandpa::Config<WithPolkadotGrandpa> for Runtime {
 	type BridgedChain = bp_darwinia::DarwiniaLike;
 	type HeadersToKeep = PolkadotHeadersToKeep;
-	type MaxBridgedAuthorities = ConstU32<100_000>;
+	type MaxBridgedAuthorities = ConstU32<4_096>;
```

Comment:
@boundless-forest Please help check if this needs a migration. I'll consider revert this change.